src/gobstuf/rest/brp/base_view.py

```diff
@@ -302,6 +302,20 @@ def _build_response(self, response_obj: StufMappedResponse, **kwargs):
             }
         }, {})
 
+    def _transform_query_parameter_value(self, value: str):
+        if not value:
```

Comment:
Just a general remark, can you add docstrings for the methods?

Reply:
You're a bit too quick. Was already on it, saw that I forgot some important ones

Reply:
Thanks!